Symfony/CS/ConfigInterface.php

```diff
@@ -59,6 +59,8 @@ public function getFixers();
      * Sets the root directory of the project.
      *
      * @param string $dir The project root directory
+     *
+     * @return Config The same instance
```

Comment:
@keradus `return $this` _helps_ IDE for typehinting. For IDEs, which can't recognize it, right annotation should be `return ConfigInterface|$this`